### PR TITLE
UXUI-247 - Removing dev folder and moving to component specific less

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do?
- Removes the `dev` folder for each component as it's really not required
- Move each less file into the specific components code directory
- Updates `readme.md` of each component to remove the `ui-` prefix
- Improves legibility of docs by adding spacing after a demo-box

#### What unit or integration tests does this PR have?
none

#### What selenium tests does this PR have?
none

#### How should a developer review this?
- Take a look at the code it should all be fairly straightforward
- Manually test as below

#### How should this be manually tested?
- `npm install`
- `grunt docs`
- check that the components in the docs still look great.

#### Any background context you want to provide?
- the `dev` folder is created by the component generator, we will put in a separate pr to remove this from the generator as its not required.

#### What are the relevant tickets?
https://holidayextras.jira.com/browse/UXUI-247
#### What gif best describes how you feel about this work?
clean and tidy

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [ ] I've checked there is appropriate unit, selenium and regression test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this cross-browser/device for visual changes.
- [x] I've checked any significant visual changes with a member of the UX/UI pod.
- [x] I've checked this work against the requirements of the Jira.
- [x] I've manually checked this work against any dependent systems.
- [ ] I've added and updated translations for all supported languages.
- [x] I've informed those who need to know of any unforeseen impact of this work (database migrations, deployment procedure changes).
- [x] I've checked that there are no negative speed or performance impacts from my work.
- [ ] I've added appropriate tracking.
- [x] I am ready for this to be code reviewed, merged and tested.
- [x] I've thought about writing a blog post about this.

#### IP or Developer Review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] +1 from me!

#### Software Engineer or Developer review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] +1 from me!

#### Software Engineer or project guru review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by rahulpatel** [comment]
Taking SE/Dev review...

File: src/less/toolkit.less [hunk 1]
```diff
@@ -4,6 +4,6 @@
 
 @import "_variables";
 
-@import "_button";
-@import "_flag";
-@import "_tile";
+@import "../components/button/code/less/_button";
+@import "../components/button/code/less/_flag";
```
Comment (rahulpatel) [inline src/less/toolkit.less]: You're importing the flag less from the button component? Same for the tile.

Could you simplify this by creating a helper function that'll return you the full path provided that you tell it what component you want the path for? (I hope that makes sense)

**Comment by stuartstobie** [comment]
I'll take the IP slot.

File: src/less/toolkit.less (hunk 1 above)
Comment (jodiedoubleday) [inline src/less/toolkit.less]: I'll do this function later on as we get more components.

**Comment by rahulpatel** [comment]
:+1:

**Comment by stuartstobie** [comment]
Still looks good :+1:

**Comment by hxjamesmcneil** [comment]
Taking SE/Guru

**Comment by hxjamesmcneil** [comment]
+1

**Pull request merged**